src/client/components/terminal.ts

```diff
@@ -373,6 +394,50 @@ export class TerminalView extends LitElement {
     }
   }
 
+  #resizeTerminal(rows?: number) {
+    if (rows !== undefined) { this.rows = rows }
+    return this.fitAddon?.fit(this.rows)
+  }
+
+  #createResizeHandle(): HTMLElement {
+    const dragHandle = document.createElement('div')
+    dragHandle.setAttribute('class', 'xterm-drag-handle')
+
+    let dragState: {
+      initialClientY: number
+      initialRows: number
+    } | undefined
+
+    dragHandle.addEventListener('mousedown', (e) => {
+      dragState = {
+        initialClientY: e.clientY,
+        initialRows: this.rows
+      }
+    })
+
+    window.addEventListener('mouseup', () => {
+      if (dragState === undefined) { return }
+      dragState = undefined
+    })
+
+    window.addEventListener('mousemove', (e) => {
```

Comment:
It's a good idea to remove the event listener at **disconnectedCallback** same applies to **mouseup**

Reply:
Yep good call 👍